operators/advanced_network/CMakeLists.txt

```diff
@@ -50,8 +47,8 @@ set(DPDK_EXTRA_LIBS -Wl,--no-whole-archive -lmlx5 -libverbs -pthread -lnuma -ldl
 
 pkg_check_modules(DPDK REQUIRED libdpdk)
 
-target_include_directories(advanced_network_rx INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
-target_include_directories(advanced_network_tx INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
+# target_include_directories(advanced_network_rx INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
@cliffburdick is it still useful to leave these here?

Reply:
Fixed